### ISSUE TITLE
cilium clustermesh status doesn't work when Clustermesh is installed via Helm

### ISSUE THREAD

**Issue by samueltorres** [cilium/cilium-cli#427]
When Clustermesh is installed via Helm, the ` cilium clustermesh status` command doesn't run correctly.

This command fails because of a mismatch on the clustermesh certificates secret names defined on cilium-cli and Helm Chart.

cilium-cli
```go
ClusterMeshServerSecretName           = "clustermesh-apiserver-server-certs"
ClusterMeshAdminSecretName            = "clustermesh-apiserver-admin-certs"
ClusterMeshClientSecretName           = "clustermesh-apiserver-client-certs"
```

Helm Chart
```yaml
  name: clustermesh-apiserver-server-cert
```
https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/clustermesh-apiserver-server-secret.yaml#L8

```yaml
  name: clustermesh-apiserver-admin-cert
```
https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/clustermesh-apiserver-admin-secret.yaml#L8

```yaml
  name: clustermesh-apiserver-client-cert
```
https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/clustermesh-apiserver-client-secret.yaml#L8

Its better to fix this on the Helm Chart right ?

**Comment by tklauser** [comment]
This looks like an oversight in cilium-cli. I think we should fix this here rather than Cilium's Helm chart.

**Comment by samueltorres** [comment]
Thanks for the feedback :slightly_smiling_face: What happens to the clusters that were set up using the current version ? Should we make it retro-compatible, like only pick up the `certs` if there is no `cert` secret ?

**Comment by tklauser** [comment]
Yes, I think making it compatible to clusters set up using the `-certs` secret would be preferrable.

**Comment by samueltorres** [comment]
I'll take care of this then.

**Comment by tklauser** [comment]
Great, thanks @samueltorres!

**Comment by life1347** [comment]
Hello, I also encountered this problem. Is there any chance to get this fix? 
BTW, is there any cluster mesh setup guide with helm? Thanks

**Comment by tklauser** [comment]
Fixed by #1056
